### ISSUE TITLE
[Accessibility], WCAG 3.3.2. Input fields are missing visible labels or instructions

### ISSUE THREAD

**Issue by Eiskore** [NatLibFi/Skosmos#1972]
## URL address of the page where you encountered the problem
https://test.dev.finto.fi/en/

## Description of the problem
 The "Yhteishaku sanastoista" exbandable button opens the menus “1. Valitse sanasto“ and “2. Valitse kieli” and “3. Kirjoita hakutermi”. The visible label /instructions about the element's purpose for these three elements receiving the input are disappearing. Labels/instructions disappear when a value is selected from the menus or text is entered in the search field.

**Fix:**
All elements that receive input must have visible labels at all times, so that users know what the field's meaning was even after selecting the value, and so that the field's visible label and the accessible name still match each other (important for speech control).

Icons and images can also be used as labels or instructions, if they are descriptive enough. The magnifying glass symbol is used in search fields and is understandable if you can search for anything in the field. However, the search function in question is limited to searching for terms only, so the displayed instruction " Kirjoita hakutermi " is essential information that should be displayed in the context of the field at all times.

It would work best if a visible label is added above all input elements.

Severity: moderate
Priority: medium

## Additional information (e.g. screenshots) about the problem

<img width="1004" height="106" alt="Image" src="https://github.com/user-attachments/assets/99be9ce4-f3e0-4693-b222-7caad6ea21fd" />

## The browser you used when the problem appeared
Chrome, Firefox, Edge

**Comment by tinskuu** [comment]
This issue is fixed.
